tests/test_routes.py

```diff
@@ -165,3 +167,23 @@ def test_delete_payment_method(self):
         # make sure they are deleted
         # response = self.client.get(f"{BASE_URL}/{test_payment_method.id}")
         # self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_list_payment(self):
+        """It should List all Payment Method"""
+        test_payment_method = CreditCardFactory()
+        test_payment_method.create()
+        test_payment_method1 = CreditCardFactory()
+        test_payment_method1.create()
+        response = self.client.get(BASE_URL)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
+        self.assertEqual(len(data), 2)
+
+    def test_get_payment(self):
+        """It should Get a single Payment"""
+        test_payment_method = CreditCardFactory()
+        test_payment_method.create()
+        response = self.client.get(f"{BASE_URL}/{test_payment_method.id}")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
+        self.assertEqual(data["name"], test_payment_method.name)
```

Comment:
also, could you throw in a test that checks the filtering for `name` field? that will increase the test coverage as well I believe since it's not covered atm